app/ml.py

```diff
@@ -492,4 +492,43 @@ async def get_weather_conditions(city: City):
     city = validate_city(city)
     weather_cond = await select_weather_conditions(city)
     
-    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
\ No newline at end of file
+    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
```

Comment:
I changed weather_temperature back to weather_conditions (which was the original). I didn't see a reason to change change this to temperature plus it would have throw an error when using this function. I'm assuming a typo?